collectors/cgroups.plugin/cgroup-network.c

```diff
@@ -479,7 +479,18 @@ void call_the_helper(pid_t pid, const char *cgroup) {
     info("running: %s", command);
 
     pid_t cgroup_pid;
-    FILE *fp = mypopene(command, &cgroup_pid, environment);
+    FILE *fp;
+
+    // fp = mypopene(command, &cgroup_pid, environment);
+    if(cgroup) {
+        (void)custom_popene(&cgroup_pid, environment, POPEN_FLAG_CREATE_PIPE | POPEN_FLAG_CLOSE_FD, &fp, PLUGINS_DIR "/cgroup-network-helper.sh", "cgroup-network-helper.sh", "--cgroup", cgroup, NULL);
+    }
+    else {
+        char buffer[100];
```

Comment:
This size is not a problem, because according official documentation for `sprintf` the function will cut the final text, but we are breaking a pattern here, please, either add `1` to `buffer` or use `99` for `sprintfz`.